README.md

```diff
@@ -1,37 +1,24 @@
 # Auth0.Android
 
-[![CircleCI](https://img.shields.io/circleci/project/github/auth0/Auth0.Android.svg?style=flat-square)](https://circleci.com/gh/auth0/Auth0.Android/tree/master)
+[![Maven Central](https://img.shields.io/maven-central/v/com.auth0.android/auth0.svg?style=flat-square)](https://search.maven.org/artifact/com.auth0.android/auth0)
 [![Coverage Status](https://img.shields.io/codecov/c/github/auth0/Auth0.Android/master.svg?style=flat-square)](https://codecov.io/github/auth0/Auth0.Android)
+[![CircleCI](https://img.shields.io/circleci/project/github/auth0/Auth0.Android.svg?style=flat-square)](https://circleci.com/gh/auth0/Auth0.Android/tree/master)
 [![License](https://img.shields.io/:license-mit-blue.svg?style=flat-square)](https://doge.mit-license.org/)
-[![Maven Central](https://img.shields.io/maven-central/v/com.auth0.android/auth0.svg?style=flat-square)](https://search.maven.org/artifact/com.auth0.android/auth0)
 [![javadoc](https://javadoc.io/badge2/com.auth0.android/auth0/javadoc.svg)](https://javadoc.io/doc/com.auth0.android/auth0)
 
-Easily integrate Auth0 into Android apps. Add **login** and **logout**, store **credentials** securely, and access **user information**.
-
-## Table of contents
-
-- [Requirements](#requirements)
-- [Installation](#installation)
-   + [Permissions](#permissions)
-- [Getting Started](#getting-started)
-   * [Authentication with Universal Login](#authentication-with-universal-login)
-   * [Clearing the session](#clearing-the-session)
-- [Next steps](#next-steps)
-   * [Authentication API](#authentication-api)
-   * [Management API](#management-api)
-   * [Token Validation](#token-validation)
-   * [Organizations](#organizations)
-- [Credentials Manager](#credentials-manager)
-   * [Basic](#basic)
-   * [Encryption enforced](#encryption-enforced)
-- [Networking client customization](#networking-client-customization)
-- [FAQ](#faq)
-- [Proguard](#proguard)
-- [What is Auth0?](#what-is-auth0)
-- [Issue Reporting](#issue-reporting)
-- [License](#license)
-
-## Requirements
+📚 [Documentation](#documentation) • 🚀 [Getting Started](#getting-started) • ⏩ [Next Steps](#next-steps) • ❓ [FAQs](https://github.com/auth0/auth0.android/blob/main/FAQ.md) • 💬 [Feedback](#feedback)
```

Comment:
Leftover next steps link I think, also I don't think the FAQs needs to be linked here if it's included in the `Documentation` section below

```suggestion
📚 [Documentation](#documentation) • 🚀 [Getting Started](#getting-started) • 💬 [Feedback](#feedback)
```

Reply:
Hey @ewanharris I was trying to be consistent with the Figma design. We do have FAQ and API Reference. Since we didn't have API Reference in the same page as Next, I added "Next Steps". What do you think about that?

![image](https://user-images.githubusercontent.com/15910425/197716433-6446c917-664d-4277-a6c5-d693b637b9d6.png)

Reply:
Looks like what I used is an older design on the same page. Committing this suggestion :)